doc/data/messages/i/invalid-sequence-index/details.rst

```diff
@@ -1 +1 @@
-Be careful with [True] or [False] as sequence index, since True is represnted as 1 and False as 0, so it will work.
+Be careful with ``[True]`` or ``[False]`` as sequence index, since ``True`` is represented as 1 and ``False`` as 0, so it will work.
```

Comment:
```suggestion
Be careful with ``[True]`` or ``[False]`` as sequence index, since ``True`` and  ``False``  will be respectively
be evaluated as ``1`` and ``0`` and will bring the second element of the list and the first without erroring.
```